<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriggerItemWidget</class>
 <widget class="QWidget" name="TriggerItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="textTb_">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="graphTb_">
       <property name="text">
        <string>graphics</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="dependTb_">
       <property name="text">
        <string>Dependencies</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MessageLabel" name="messageLabel_" native="true"/>
   </item>
   <item>
    <widget class="TriggerBrowser" name="textBrowser_" native="true"/>
   </item>
   <item>
    <widget class="TriggerView" name="triggerView_"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TriggerView</class>
   <extends>QGraphicsView</extends>
   <header>TriggerView.hpp</header>
  </customwidget>
  <customwidget>
   <class>TriggerBrowser</class>
   <extends>QWidget</extends>
   <header>TriggerBrowser.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
